buildSrc/src/test/kotlin/com/datadog/android/core/internal/utils/MiscUtils.kt

```diff
@@ -4,7 +4,7 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-package com.example.model
+package  com.datadog.android.core.`internal`.utils
```

Comment:
```suggestion
package com.datadog.android.core.internal.utils
```

Reply:
Sorry I need to get back to this as I tested it based on my schemas and we are missing some things:

Given this example:

```
{
    "$schema": "http://json-schema.org/draft-07/schema",
    "$id": "record-schema.json",
    "title": "Record",
    "type": "object",
    "description": "Schema of a Session Replay Record.",
    "oneOf": [
        {
            "$ref": "metadata-record-schema.json"
        },
        {
            "$ref": "focus-record-schema.json"
        },
        {
            "$ref": "view-end-record-schema.json"
        }
    ]
}
```

It will generate the sealed Classes but it will create duplicates for each of the referenced schema in separated classes.

I also tried to reference this `record-schema.json` from an array inside another schema:

```
 "records": {
                    "type": "array",
                    "items": {
                        "$ref": "record-schema.json"
                    },
                    "description": "The records contained by this Segment.",
                    "readOnly": true
                }
```
It crashes in this moment, in this case this array should reference the sealed Class.